language-descriptions/spanish-language-profile.md

```diff
@@ -149,6 +149,7 @@ As an international language with millions of native speakers, Spanish has wide
 - [Podcastsinspanish.org](http://www.podcastsinspanish.org/)
 - [Spanish obsessed](https://spanishobsessed.com/)
 - [conjugacion.es](http://www.conjugacion.es/)
+- [Spanish Resources to become fluent](https://tutorful.co.uk/blog/learning-spanish-the-best-resources-to-become-fluent/)
```

Comment:
This is a link to an article that you wrote. Please sign up for an account on the forum (if you haven't already) and discuss your proposed change there. If other people think it's a worthwhile link, then I'll be happy to add it.